daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -963,9 +965,11 @@ func (devices *DeviceSet) initDevmapper(doInit bool) error {
 
 	// https://github.com/docker/docker/issues/4036
 	if supported := devicemapper.UdevSetSyncSupport(true); !supported {
-		logrus.Warnf("Udev sync is not supported. This will lead to unexpected behavior, data loss and errors")
+		logrus.Errorf("Udev sync is not supported. This will lead to unexpected behavior, data loss and errors. For more information, see http://docs.docker.com/reference/commandline/cli/#daemon-storage-driver-option")
```

Comment:
... k
Though this should force a redirect if thats the case, because this is
where google sent me ...

On Tue, Apr 7, 2015 at 1:23 PM, Tibor Vass <notifications@github.com> wrote:

> In daemon/graphdriver/devmapper/deviceset.go
> <https://github.com/docker/docker/pull/11412#discussion_r27899608>:
>
> > @@ -963,9 +965,11 @@ func (devices *DeviceSet) initDevmapper(doInit bool) error {
> >
> >  	// https://github.com/docker/docker/issues/4036
> >  	if supported := devicemapper.UdevSetSyncSupport(true); !supported {
> > -		logrus.Warnf("Udev sync is not supported. This will lead to unexpected behavior, data loss and errors")
> > +		logrus.Errorf("Udev sync is not supported. This will lead to unexpected behavior, data loss and errors. For more information, see http://docs.docker.com/reference/commandline/cli/#daemon-storage-driver-option")
>
> https
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/11412/files#r27899608>.
>